### PR TITLE
unlock trusttokens

### PR DESCRIPTION
**Pull request by hal909**

### CODE REVIEW

File: contracts/Unlock.sol [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+     */
+    function deliver(uint256 id, address beneficiary) public {
+        address holder = unlockOperations[id].recipient;
+        require(msg.sender == holder);
+        require(unlockOperations[id].activation <= now);
+        emit UnlockClaimed(id, beneficiary);
+        emit Transfer(holder, address(0), id);
+        balanceOf[holder]--;
+        burnCount++;
+        unlockOperations[id].recipient = address(0);
```
Comment (Marik-D) [inline contracts/Unlock.sol:226]: This can be extracted as `_burn`

File: contracts/Unlock.sol [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+
+    function symbol() public view returns (string memory) {
+        return token().symbol();
+    }
+
+    /**
+     * @dev Deliver vault tokens to address.
+    **/
+    function deliver(address _to, uint256 _amount) external onlyOwner {
+        token().transferFrom(address(this), _to, _amount);
```
Comment (Marik-D) [inline contracts/Unlock.sol:92]: Why not
```
        token().transfer(_to, _amount);
```

File: test/UnlockTrustTokens.test.js [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+            await assertRevert(this.unlock.claim(0, {from: account1}))
+            await assertRevert(this.unlock.deliver(0, account2, {from: account1}))
+        })
+    })
+    describe('mint trusttokens', function() {
+        it('mint', async function() {
+            await this.vault.claimTokenOwnership({from: owner})
+            await this.vault.mintTrustTokens({from: owner});
+            assert.equal(TRUST_TOKEN_MINT_AMOUNT, await this.vault.vaultBalance.call())
+        })
```
Comment (Marik-D) [inline test/UnlockTrustTokens.test.js:62]: Add cannot mint twice test

**Comment by krzysztof-jelski** [comment]
Unlock removed as of recent decision.

**Pull request closed without merging**